docs/aws_tools.md

```diff
@@ -10,67 +10,52 @@ nav_order: 3
 
-{: .more }
-<https://aws.amazon.com/console>{:target="_blank"}
-
 ## AWS CLI
 
-The AWS Command Line Interface (CLI) is a powerful tool that enables users to interact with AWS services through a command-line interface. It provides a unified interface to perform various operations, automate tasks, and manage AWS resources. The AWS CLI offers extensive functionalities, including resource management, service provisioning, and interaction with AWS APIs.
+The [AWS Command Line Interface (CLI)](https://aws.amazon.com/cli){:target="_blank"} is a powerful tool that enables users to interact with AWS services through a command-line interface. It provides a **unified interface to perform various operations**, automate tasks, and manage AWS resources. The AWS CLI offers extensive functionalities, including resource management, service provisioning, and interaction with AWS APIs.
 
-The AWS CLI is used for automating testing workflows and tasks using scripts or CI/CD pipelines, interacting with AWS services programmatically for testing purposes, and managing and configuring AWS resources required for testing.
+The AWS CLI is used for **automating testing workflows and tasks using scripts** or CI/CD pipelines, interacting with AWS services programmatically for testing purposes, and managing and configuring AWS resources required for testing.
 
 The AWS CLI is particularly useful for tasks related to integration testing, system testing, performance testing, security testing, and regression testing.
 
-{: .more }
-<https://aws.amazon.com/cli>{:target="_blank"}
-
 ## AWS SDK
 
-The AWS Software Development Kit (SDK) provides programming interfaces and libraries for multiple programming languages. It allows engineers to interact with AWS services programmatically, integrating AWS functionality into their applications and tests. The AWS SDK provides a range of features and utilities to simplify the development and testing process, including authentication, request signing, and service-specific operations.
+The [AWS Software Development Kit (SDK)](https://aws.amazon.com/sdk-for-javascript/){:target="_blank"} provides programming interfaces and libraries for multiple programming languages. It allows engineers to **interact with AWS services programmatically**, integrating AWS functionality into their applications and tests. The AWS SDK provides a range of features and utilities to simplify the development and testing process, including authentication, request signing, and service-specific operations.
 
 The AWS SDK is used for writing custom test scripts and scenarios to validate application behavior, simulating interactions with AWS services for testing purposes, and automating the execution of test cases and scenarios using the AWS SDK.
 
 The AWS SDK is essential for tasks associated with unit testing, integration testing, system testing, and performance testing.
 
-{: .more }
-<https://www.npmjs.com/package/aws-sdk>{:target="_blank"}
-
 ## AWS CDK
```

Comment:
As Both AWS CDK and AWS SAM were mentioned, I think it is worth mentioning AWS CF as well. Maybe not in a section of its own, but for example under aws-cdk, with something like:

`[AWS CloudFormation (CF)](https://aws.amazon.com/cloudformation/) is an AWS service that automates the creation and management of cloud resources using declarative templates (ex. JSON and YAML files). It simplifies infrastructure provisioning, allows for consistent and efficient resource management, and supports updates and deletions of resource stacks.`

I think this would be good as both CDK and SAM are generating CloudFormation templates to create the stacks. Also it was mentioned on line 57, elaborating more on it would be best.

Reply:
As CF is not an individual tool (that can be installed) but a service of AWS, I'd not list it here. We can add it to the AWS services as a separate page, or add it to the Testing Types / Infrastructure testing page. What do you think?